stable/mlrun-kit/README.md

```diff
@@ -1,143 +1,161 @@
-[**examples/mlrun_basics.ipynb**](https://github.com/mlrun/mlrun/blob/master/examples/mlrun_basics.ipynb) notebook.
-
-> **Note:**
-> - You can change the ports by providing values to the helm install command.
-> - You can add and configure a k8s ingress-controller for better security and control over external access.
-
-
-## Advanced Chart Configuration
-Configurable values are documented in the `values.yaml`, and the `values.yaml` of all sub charts. 
-Override those [in the normal methods](https://helm.sh/docs/chart_template_guide/values_files/).
-
-## Uninstalling the Chart
-```bash
-$ helm --namespace mlrun uninstall my-mlrun
-```
-
-#### Note on terminating pods and hanging resources
-It is important to note that this chart generates several persistent volume claims and also provisions an NFS
-provisioning server, to provide the user with persistency (via pvc) out of the box.
-Because of the persistency of PV/PVC resources, after installing this chart, PVs and PVCs will be created,
-And upon uninstallation, any hanging / terminating pods will hold the PVCs and PVs respectively, as those
-Prevent their safe removal.
-Because pods stuck in terminating state seem to be a never-ending plague in k8s, please note this,
-And don't forget to clean the remaining PVCs and PVs
-
-Handing stuck-at-terminating pods:
-```bash
-$ kubectl --namespace mlrun delete pod --force --grace-period=0 <pod-name>
-```
-
-Reclaim dangling persistency resources:
-
-| WARNING: This will result in data loss! |
-| --- |
-
-```bash
-# To list PVCs
-$ kubectl --namespace mlrun get pvc
-...
-
-# To remove a PVC
-$ kubectl --namespace mlrun delete pvc <pvc-name>
-...
-
-# To list PVs
-$ kubectl --namespace mlrun get pv
-...
-
-# To remove a PVC
-$ kubectl --namespace mlrun delete pvc <pv-name>
-
-# Remove hostpath(s) used for mlrun (and possibly nfs). Those will be created, by default under /tmp, and will contain
-# your release name, e.g.:
-$ rm -rf my-mlrun-mlrun-kit-mlrun
-...
-```
-
-### Using Kubeflow Pipelines
-
-MLRun enables you to run your functions while saving outputs and artifacts in a way that is visible to Kubeflow Pipelines.
-If you wish to use this capability you will need to install Kubeflow on your cluster.
-Refer to the [**Kubeflow documentation**](https://www.kubeflow.org/docs/started/getting-started/) for more information.
+# MLRun Kit: MLRun Open Source Kit for MLOps
+
+This Helm charts bundles open source software stack for advanced ML operations
+
+## Chart Details
+
+The Open source MLRun kit chart includes the following stack:
+
+* Nuclio - https://github.com/nuclio/nuclio
+* MLRun - https://github.com/mlrun/mlrun
+* Jupyter - https://github.com/jupyter/notebook (+MLRun integrated)
+* NFS - https://github.com/kubernetes-retired/external-storage/tree/master/nfs
+* MPI Operator - https://github.com/kubeflow/mpi-operator
+* Minio - https://github.com/minio/minio/tree/master/helm/minio

```

Comment:
I think we should seperate NFS and Minio from this list and add it under **Supported storage solutions** title, feel free to offer different title name I am not quite sure about the naming.